src/Symfony/Component/Config/Tests/Util/XmlUtilsTest.php

```diff
@@ -221,7 +221,7 @@ public function testLoadWrongEmptyXMLWithErrorHandler()
             error_reporting($errorReporting);
         }
 
-        if (\LIBXML_VERSION < 20900) {
```

Comment:
This is wrong, we should check the version of the LIB when disabling the entity loader (calling with `true`)

same below